src/rewrite.ml

```diff
@@ -106,9 +197,10 @@ let handle_rewrite : term -> unit = fun t ->
         fatal_no_pos "Rewrite expects a goal of the form “P t” (found [%a])."
           pp g.g_type
   in
-
-  (* Build the predicate by matching [l] in [g_term]. *)
-  (* TODO keep track of substitutions in the first instance of l in g. *)
+  (***************************************************************************)
```

Comment:
It is best not to use this separation style, for uniformity reasons. You can do as above: skip a line, and add a comment explaining what happens in this small block of code.